pkg/urbit/noun/events.c

```diff
@@ -255,9 +255,9 @@ u3e_fault(void* adr_v, c3_i ser_i)
 
   c3_w* adr_w = (c3_w*) adr_v;
 
-  if ( (adr_w < u3_Loom) || (adr_w >= (u3_Loom + u3a_words)) ) {
+  if ( (adr_w < u3_Loom) || (adr_w >= (u3_Loom + u3C.wor_i)) ) {
```

Comment:
This line is the only change that I find potentially worrisome. We could mark `u3C.wor_i` as `volatile`, but I don't think we should have to since it's only ever set once at initialization.

Reply:
I agree that this should be fine. Unless there's a `SEGFAULT` between `u3e_live()` and `u3m_pave()` in `u3m_boot()`, the fault handler won't run until after `u3C.wor_i` has been initialized. We already access `u3P.dit_w` in the fault handler, and that use case is read/write, not read-only like `u3C.wor_i`.